src/core/App/index.js

```diff
@@ -7,7 +7,7 @@ export const App = () => {
     <ThemeProvider theme={theme}>
```

Comment:
czy nie potrzeba zaimportować jeszcze theme?

Reply:
Yes, it necessary.

Reply:
Yes, I will do that right now.